### PR TITLE
Removes passive income from the Security Budget

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request

There is no good way to balance around their budget. Due to the Economy subsystem not respecting the universal laws of time it will be permanently out of wack. This is better than playing whack-a-mole with 100k sec budgets 30 minutes into the game.

They now start with 10k, you can deposit and withdraw but no money is added on the Economy tick

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Sec budget gets 10k roundstart and no more

### What general grouping does this PR fall under? 
Cargo rebalancing

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
0

# Changelog

:cl:  
rscdel: The Security budget now only gets a flat sum of money each shift.
/:cl:

### CODE REVIEW

**Comment by adamsong** [comment]
Is the budget card not required for the warden's paystand?

**Comment by Bibby0110** [comment]
> Is the budget card not required for the warden's paystand?

Good point, I'll replace it

**Comment by Marmio64** [comment]
sec deserve some kind of budget though. How about a card that just has like 10k on it or something, or even just some plain old spacecash in the card's place. Removing the budget entirely would just make sec's lives way harder than it has to be, and it can already be a nightmare to try to survive against a cult or something so long as that cult has at least one good player.

**Comment by Bibby0110** [comment]
> sec deserve some kind of budget though. How about a card that just has like 10k on it or something, or even just some plain old spacecash in the card's place. Removing the budget entirely would just make sec's lives way harder than it has to be, and it can already be a nightmare to try to survive against a cult or something so long as that cult has at least one good player.

That doesn’t really have much to do with the budget but more to do with the skill of security, but you’re welcome to code in an alternative yourself. This is just horribly broken as it stands and I don’t personally see it needing a replacement, but as I said you’re welcome to PR your own alternative :)

**Comment by Bibby0110** [comment]
> sec deserve some kind of budget though. How about a card that just has like 10k on it or something, or even just some plain old spacecash in the card's place. Removing the budget entirely would just make sec's lives way harder than it has to be, and it can already be a nightmare to try to survive against a cult or something so long as that cult has at least one good player.

On further thought you're right, it's been locked to 10k (2x regular starting amount).

**Comment by Bibby0110** [comment]
I'm going to do an epic drift and re-open this PR since I changed it to be something totally different 😄
